src/Makefile.am

```
@@ -234,6 +234,8 @@ squid_SOURCES = \
 	carp.cc \
 	cbdata.cc \
 	cbdata.h \
+	ClientActiveRequests.h \
+	ClientActiveRequests.cc \
```

This list is inconsistent, so I do not insist on this change, but I believe this list was meant to be alphabetized:

```suggestion
	ClientActiveRequests.cc \
	ClientActiveRequests.h \
```

Same for other lists.

---

We are not very consistent there at all. Changing, but something more generic may be needed.
I'll see if I can cook up some script or similar.

---

> I'll see if I can cook up some script or similar.

Your call. FWIW, I would not spend time on marginal improvements here -- any decent editor can sort a few lines alphabetically, which is sufficient for most cases. You are right that the current lists is grossly inconsistent and terrible from maintenance point of view, but I suspect that the proper solution lies in a complete revamping on how we manage these lists, and the solution space might even include abandoning automake.

---

You are correct. At the same time, this would be inconvenient for stubs; they follow a good naming convention and it'd be nice to sort them close to the header files they refer to.

---

Yes, that so called "good" naming convention is not so good for the purpose of listing dependencies in a consistent human-friendly way. We have to pick our poison.

Again, I know that this area is currently very messy, and I do not insist on this minor mess-reducing change.